homeassistant/components/tplink/switch.py

```diff
@@ -37,10 +37,51 @@ async def async_setup_entry(
             entities.append(SmartPlugSwitch(child, coordinator))
     else:
         entities.append(SmartPlugSwitch(device, coordinator))
+        entities.append(SmartPlugSwitch(SmartDeviceLedWrapper(device), coordinator))
 
     async_add_entities(entities)
 
 
+class SmartDeviceLedWrapper(SmartDevice):  # type: ignore
```

Comment:
```suggestion
class SmartDeviceLedWrapper(CoordinatedTPLinkEntity, SwitchEntity):
```

Reply:
Hey there! You just crossed my wire, as I'm working on updating this at the moment (hence the WIP tag in the title)
Thanks for taking a look!